exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/diagnosiskey/v1/TemporaryExposureKeyRequest.java

```diff
@@ -15,31 +17,44 @@
  * https://developers.google.com/android/exposure-notifications/verification-system#metadata
  */
 public final class TemporaryExposureKeyRequest {
-    /** Key of infected user: the byte-array, Base64-encoded **/
+    /**
+     * Key of infected user: the byte-array, Base64-encoded
+     **/
     public final String keyData;
-    /** Varying risk associated with a key depending on diagnosis method **/
+    /**
+     * Varying risk associated with a key depending on diagnosis method
+     **/
     public final int transmissionRiskLevel;
-    /** The interval number since epoch for which a key starts **/
+    /**
+     * The interval number since epoch for which a key starts
+     **/
     public final int rollingStartIntervalNumber;
-    /** Increments of 10 minutes describing how long a key is valid **/
+    /**
+     * Increments of 10 minutes describing how long a key is valid
+     **/
     public final int rollingPeriod;
-    /** Set of visited countries in ISO-3166 alpha-2 format **/
-    public final Set<String> visitedCountries;
-    /** Consent to share data with efgs **/
+    /**
+     * Set of visited countries in ISO-3166 alpha-2 format
+     **/
+    public final Map<String, Boolean> visitedCountries;
```

Comment:
Move to request-level, not key-level, as they're given just once.

Reply:
Ah, you are correct. It was already there, so didn't double check think that. Good point. I'll fix this.